### ISSUE TITLE
twilio_conversations_push_credential_apn fails to unmarshal the string response for Sandbox from twilio API

### ISSUE THREAD

**Issue by bbl232** [RJPearson94/terraform-provider-twilio#66]
## Technical Details

- OS: Linux_AMD64
- Terraform Version: 0.12.30
- Provider Version 0.20.0

## Affected Resources

- twilio_conversations_push_credential_apn

## Describe the bug

Looks like the twilio API responds with a string "True" or "False" for the Sandbox value, which is mismatched with the expected bool in the sdk you've written for this provider. This results in the following error.

Failed to create conversations credential: json: cannot unmarshal string into Go struct field CreateCredentialResponse.sandbox of type bool

## Steps to Reproduce

Steps to reproduce the behaviour:

1. Create resource

## Expected behaviour

Resource created and able to proceed with other resource creation

## Logs

Failed to create conversations credential: json: cannot unmarshal string into Go struct field CreateCredentialResponse.sandbox of type bool

**Comment by bbl232** [comment]
Upon further review I believe this is just an issue in the api definition for conversations/credentials response in the SDK you wrote to accompany this tf provider

**Comment by RJPearson94** [comment]
Hi @bbl232,

Yes, you are correct the issue is with the underlying go SDK. Thanks for raising a PR against the SDK, I will take a look at getting that merged and update the provider.

Thanks
Rob

**Comment by RJPearson94** [comment]
Hi @bbl232, the changes to resolve this issue are already in the main branch. I am just working on resolving #65 and then this will be released at the same time.

**Comment by RJPearson94** [comment]
Hi @bbl232 

Apologies for the delay, the fix for this issue has been included in v0.21.0, which is now available in the Terraform Registry.

Please update your Terraform configuration to include the following

```hcl
terraform {
  required_providers {
    twilio = {
      source = "RJPearson94/twilio"
      version = ">= 0.21.0"
    }
  }
} 
```

Kind Regards
Rob
